libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/binder/infopanel/InfoPanelBinderTest.kt

```diff
@@ -22,36 +21,83 @@ import org.robolectric.RobolectricTestRunner
 internal class InfoPanelBinderTest {
 
     private lateinit var ctx: Context
-    private lateinit var sut: InfoPanelBinder
+    private lateinit var sut: StubInfoPanelBinder
```

Comment:
```suggestion
    private lateinit var sut: InfoPanelBinder
```

Reply:
This is correct. We need to be able to access StubInfoPanelBinder properties.